app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -189,36 +175,38 @@ def actor
 
     ##
     # @return [#errors]
-    def create_work
-      case curation_concern
-      when ActiveFedora::Base
-        actor.create(actor_environment)
-      else
-        form = build_form
-
-        @curation_concern =
-          form.validate(params[hash_key_for_curation_concern]) &&
-          transactions['change_set.create_work']
-          .with_step_args('work_resource.add_to_parent' => { parent_id: params[:parent_id], user: current_user },
-                          'work_resource.add_file_sets' => { uploaded_files: uploaded_files, file_set_params: params[hash_key_for_curation_concern][:file_set] },
-                          'change_set.set_user_as_depositor' => { user: current_user })
-          .call(form).value!
-      end
+    def create_valkyrie_work
+      form = build_form
+      return after_create_error(form_err_msg(form)) unless form.validate(params[hash_key_for_curation_concern])
```

Comment:
Stopping the create and processing the error when validation fails is the fix for Issue #5465.